lib/matplotlib/patches.py

```diff
@@ -3315,8 +3379,8 @@ def transmute(self, path, mutation_size, linewidth):
             # This simple code will not work if ddx, ddy is greater than the
             # separation between vertices.
             _path = [Path(np.concatenate([[(x0 + ddxA, y0 + ddyA)],
-                                          path.vertices[1:-1],
-                                          [(x3 + ddxB, y3 + ddyB)]]),
+                                         path.vertices[1:-1],
+                                         [(x3 + ddxB, y3 + ddyB)]]),
```

Comment:
This change appears unnecessary.

Reply:
Without the spaces, flake8 return E128(continuation line under-indented for visual indent) for both lines.

Reply:
Strange, but 🤷‍♂️